### ISSUE TITLE
[FEATURE] Search box needs a label

### ISSUE THREAD

**Issue by emmalearnscode** [EddieHubCommunity/LinkFree#521]
### Description

For accessibility reasons the search input should have an associated label and the input field should have a name and id so that the two can be linked.

**Comment by ryan-broguiere** [comment]
Hi, could I handle this issue?

**Comment by krishguptadev** [comment]
```html
<label id="label">Text</label>
<input labelledby="label">
```

You mean something like ☝️ but in react?
